consensus/merge/src/test/java/org/hyperledger/besu/consensus/merge/TransitionProtocolScheduleTest.java

```diff
@@ -169,6 +167,7 @@ public void getByBlockNumber_returnsTimestampScheduleIfPresent() {
     verifyNoMergeScheduleInteractions();
   }
 
+  @Ignore
```

Comment:
why ignore these tests? is there a follow up task?

Reply:
this file was prob committed accidentally. Thanks @macfarla